src/components/Features/Features.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { FiFilter } from 'react-icons/fi';
+import { GrLanguage } from 'react-icons/gr';
+import { SiGotomeeting } from 'react-icons/si';
+import FeatureCard from './FeatureCard';
+
+const Features = () => {
+  return (
+    <section
+      data-testid="features-section"
+      className="bg-background dark:bg-gray-900"
+    >
+      <div className="py-8 px-4 mx-auto max-w-screen-xl sm:py-16 lg:px-6">
+        <div className="mb-8 lg:mb-16">
+          <h2
+            data-testid="features-title"
+            className="mb-4 text-4xl tracking-tight font-extrabold text-gray-900 dark:text-white"
+          >
+            Designed for your needs and kind
+          </h2>
+          <p
+            data-testid="features-paragraph"
+            className="text-gray-500 sm:text-xl dark:text-gray-400"
+          >
+            Here at Dilmun we focus on joining people together and help those
+            people that are in need.
```

Comment:
i don't think this is the main focus since we're building an e-commerce website. I suggest looking up Etsy's or Amazon's goals.

Reply:
It's donating website not e commerce i think

Reply:
It's donation and e-commerce. So yeah changing this makes sense.